app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/Routes.kt

```diff
@@ -12,13 +12,14 @@ object Routes {
     const val DevOptions = "devoptions"
     const val Snippets = "snippets"
     const val Profile = "profile"
+    const val Commits = "commits"
 
-    fun RepositoryBrowser(data: RepositoryBrowserData) =
+    fun RepositoryBrowser(data: RepositoryBrowserData = RepositoryBrowserData("{repoName}", "{folderHash}", "{folderPath}")) =
```

Comment:
Can we use these new default values in MainNavGraph for the route definition?  If so, that would be great addition.

Reply:
I can look into that, it works when doing the current route check in Compose Activity